README.md

```diff
@@ -107,7 +116,51 @@ openssl req -new -newkey rsa:4096 -x509 -sha256 -days 365 -nodes \
   -addext "subjectAltName=DNS:keycloak.example.com,DNS:glowingbear.example.com"
 ```
 
-In case of a self-signed certificate, copy the file `ssl/server.pem` to
+### Certificates signed by CA
+
+CA usually provide you with 4 files of following types:
+
+* certificate file for your hostname (eg. `cert.pem`);
+
+* private key file for your hostname (eg. `privkey.pem`);
+
+* chain file (eg. `chain.pem`) - this file contains a certificate chain of CA
+
+```bash
```

Comment:
Maybe indent this part, so that is is part of the 'chain file' bullet

Reply:
Fixed.